OpenMined/PySyft
# Configure Privacy Budget Notebook

-added rst files and images for the configure PB doc 
-added two image for the docs

## Affected Dependencies
Merges with PR #6759 
Merges with PR #6693 
Closes issue #6688 


## How has this been tested?
- Make sure the notebooks run
- Keep an eye out for formatting of code in cells
- No excess spacing on headings
- Use normal font size for main text and custom headings
- Proofread spelling and punctuation 
- Check for consistency
